cmd/checkIfStepActive.go

```diff
@@ -149,7 +136,7 @@ func addCheckStepActiveFlags(cmd *cobra.Command) {
 		"Default config of piper pipeline stages")
 	cmd.Flags().StringVar(&checkStepActiveOptions.stepName, "step", "", "Name of the step being checked")
 	cmd.Flags().StringVar(&checkStepActiveOptions.stageName, "stage", "", "Name of the stage in which contains the step being checked")
-	cmd.Flags().BoolVar(&checkStepActiveOptions.v1Active, "useV1", false, "Use new CRD-style stage configuration")
+	cmd.Flags().BoolVar(&checkStepActiveOptions.v1Active, "useV1", false, "Use new CRD-style stage configuration (deprecated)")
```

Comment:
The logic around `useV1` is removed, but the param cannot be removed now, because there are a couple of places where the param is hardcoded.

Reply:
Done [1c9138](https://github.com/SAP/jenkins-library/pull/4814/commits/1c91387eaf814df098b9a1da95983ae36af36ba2)